### PR TITLE
Add compat data for RTCSessionDescription

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/RTCSessionDescription

### CODE REVIEW

File: api/RTCSessionDescription.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true,
+            "notes": "Use with <code>-moz</code>"
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: We have a special entry for prefixes. Do it that way:
"firefox": {
   "prefix": "-moz-",
   "version_added": "true"
}

(and no note)

File: api/RTCSessionDescription.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true,
+              "notes": "Use with <code>-moz</code>"
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: No prefix, no note here (the interface is prefixed, not the member name)

File: api/RTCSessionDescription.json [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: No prefix, no note here (the interface is prefixed, not the member name)

File: api/RTCSessionDescription.json [hunk 4]
```diff
@@ -0,0 +1,151 @@
+{
+  "api": {
+    "RTCSessionDescription": {
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: Constructor + toJSON methods are missing in the subfeature set here, please add them

File: api/RTCSessionDescription.json [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "RTCSessionDescription": {
+        "__compat": {
+          "description": "<code>RTCSessionDescription</code> constructor.",
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: No final dot, please

File: api/RTCSessionDescription.json [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "toJSON": {
+        "__compat": {
+          "description": "<code>toJSON</code> method.",
```
Comment (teoli2003) [inline api/RTCSessionDescription.json]: Not needed, only for constructors.

File: api/RTCSessionDescription.json (hunk 5 above)
Comment (bunnybooboo) [reply api/RTCSessionDescription.json]: Ha I actually anguished over that!

**Pull request merged**